java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/HybridseUtil.scala

```diff
@@ -272,7 +276,15 @@ object HybridseUtil {
     extraOptions += ("coalesce" -> parseOption(getOptionFromNode(node, "coalesce"), "0", getIntOrDefault))
     extraOptions += ("create_if_not_exists" -> parseOption(getOptionFromNode(node, "create_if_not_exists"),
       "true", getBoolOrDefault))
+    extraOptions += ("is_check_schema" -> parseOption(getOptionFromNode(node, "is_check_schema"),
+      "true", getBoolOrDefault))
 
+    val storage = ctx.getConf.loadDataMode
```

Comment:
you can add an arg `loaddatamode`, no need to add ctx here

Reply:
As discussed previously,`is_check_schema` is not only applicable to `tidb`, but can also be used with other types of data sources in data loading commands. The reason for parsing this parameter in `parseOptions` is because it needs to be included in the `options` and referenced in the soft copy as well.

Reply:
> you can add an arg `loaddatamode`, no need to add ctx here. And it's better to rewrite `is_check_schema` in load data online, cuz you just check is_check_schema when tidb load. Besides, It's not good to rewrite in arg parse.

As discussed previously,`is_check_schema` is not only applicable to `tidb`, but can also be used with other types of data sources in data loading commands. The reason for parsing this parameter in `parseOptions` is because it needs to be included in the `options` and referenced in the soft copy as well.

Reply:
updated